client/src/components/layouts/Navigation.js

```diff
@@ -31,7 +32,7 @@ const Navigation = ({ users }) => {
     if (!auth.isSignedIn) {
```

Comment:
How about the checks if the user is not an admin?

Reply:
Will implement that sir

Reply:
Is this included in as sub-task or this will be a seperate PR?

Reply:
Is it fine to just create another task in Asana for a separate PR for this one sir?

Reply:
Ok. Create a separate task and PR. I'm ok with this, I will merge